kafka/consumer.go

```diff
@@ -576,6 +576,23 @@ func (c *Consumer) Committed(partitions []TopicPartition, timeoutMs int) (offset
 	return newTopicPartitionsFromCparts(cparts), nil
 }
 
+// Position returns the current consume position for the given partitions.
+// Typical use is to call Assignment() to get the partition list
+// and then pass it to Position() to get the current consume position for
+// each of the assigned partitions.
+// The conusme position is the next message to read from the partition.
+// i.e., the offset of the last message seen by the application + 1.
```

Comment:
I would rephrase as `last message consumed + 1` instead of the `last message seen by the application + 1`, since there's no guarantee the last message of the given partition was read by this application (e.g. if this application just started).

Reply:
Position() returns the last message +1 passed to this application instance, I think you are referring to Committed() which is the last committed message + 1.